apps/web/components/booking/BookingListItem.tsx

```diff
@@ -656,10 +674,60 @@ type AttendeeProps = {
 };
 
 const Attendee = ({ email, name }: AttendeeProps) => {
+  const { t } = useLocale();
+  const [noShow, setNoShow] = useState(false);
+
+  function toggleNoShow(noShow: boolean, event: React.MouseEvent<HTMLButtonElement>) {
+    // TODO: backend call to update noShow status peer attendee
+
+    // TODO: optimistically update UI, likely without useState
+    setNoShow(!noShow);
+    event.preventDefault();
+    event.stopPropagation();
+  }
+
   return (
-    <a className="hover:text-blue-500" href={`mailto:${email}`} onClick={(e) => e.stopPropagation()}>
-      {name || email}
-    </a>
+    <Dropdown>
+      <DropdownMenuTrigger asChild>
+        <button
+          onClick={(e) => e.stopPropagation()}
+          className="radix-state-open:text-blue-500 hover:text-blue-500">
+          {noShow ? (
+            <s>
+              {name || email} <EyeOff className="inline h-4" />
+            </s>
+          ) : (
+            <>{name || email}</>
+          )}
+        </button>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent>
+        {/* TODO: figure out how to close the dropdown after clicking, since we are stopping propagation due to the <Link> */}
+        <DropdownMenuItem className="focus:outline-none">
+          {/* TODO: add subject: title */}
+          <DropdownItem StartIcon={Mail} href={`mailto:${email}`} onClick={(e) => e.stopPropagation()}>
+            <a href={`mailto:${email}`}>{t("email")}</a>
+          </DropdownItem>
+        </DropdownMenuItem>
```

Comment:
Interesting - i think we can use DropdownItem and pass href in without needing to put a link inside. Let me know if this doesnt work.

```suggestion
        <DropdownMenuItem className="focus:outline-none">
          {/* TODO: add subject: title */}
          <DropdownItem StartIcon={Mail} href={`mailto:${email}`} >
           {t("email")}
          </DropdownItem>
        </DropdownMenuItem>
```
If this doesnt work we may need to implement the "asChild" prop for DropdownItem

Reply:
thats fine